test/vm.install

```diff
@@ -49,7 +49,9 @@ def vm_install(image, verbose, quick):
             # cockpit-storaged is also available in the default rawhide compose, make sure we don't pull it from there
             download_from_copr(f"packit/cockpit-project-cockpit-{cockpit_pr}", "cockpit-storaged", machine)
         else:
-            packages_to_download += " cockpit-storaged"
+            # FIXME: Download cockpi-storaged from custom COPR
+            # till https://github.com/cockpit-project/cockpit/pull/19924#event-11785480839 is released
+            download_from_copr("@cockpit/main-builds", "cockpit-storaged", machine)
```

Comment:
FTR, it landed in F38/39 but I held back 40/rawhide for this. So this is good for now.